DEV-10/ArrayConsistingOfRepeatingElements/RandomNumbersArrayCreator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace ArrayConsistingOfRepeatingElements
+{
+    /// <summary>
+    /// Class is for creating the array of random real numbers.
+    /// </summary>
+    class RandomNumbersArrayCreator
```

Comment:
Unnecessary class, better to joins such classes into one reasonable with different methods